pyproject.toml

```diff
@@ -22,3 +26,96 @@ known_first_party = ["partcad"]
 [tools.pyright]
 reportMissingTypeArgument = true  # Report generic classes used without type arguments
 strictListInference = true  # Use union types when inferring types of lists elements, instead of Any
+
+# export POETRY_MULTIPROJECT_CHECK_DIR="/tmp/.check_partcad-dev"
+# export POETRY_MULTIPROJECT_PREPARE_DIR="/tmp/.prepare_partcad-dev"
+
+[tool.poetry.plugins."poetry-multiproject-plugin"]
+cache-dir = "/tmp/.prepare_partcad-dev"
+
+[tool.poetry]
+name = "partcad-dev"
+version = "0.1.0"
+description = "Development Environment for PartCAD"
+authors = [
+  "Roman Kuzmenko <support@partcad.org>",
+  "Alexander Ilyin <alexander@ilyin.eu>"
+]
```

Comment:
- [ ] Not sure on the right approach here, need to discuss with @openvmp.

Reply:
Yeah, perhaps we need to drop all personal names everywhere. Leave the company info if there is a field for that